Sources/AWSLambdaEvents/LambdaProxyEvent.swift

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2022 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+public struct LambdaProxyEvent: Codable {
```

Comment:
Since we are only ever decoding these events, is `Decodable` conformance enough?

Reply:
iiuc this is specifically for an APIGateway proxy? If so, should we use a less generic name than `LambdaProxyEvent`?

Reply:
1. I would agree with keeping the type only to the Decodable

2. The name was suggested by Fabian, how about 'LambdaGatewayProxyEvent' ? Trying to keep the length to a minimum also